priprava/instalace-nastroju/node.md

```diff
@@ -2,6 +2,22 @@
 
 ::fig[logo Node.js]{src=assets/nodejs.svg size=10}
 
-Nyní je potřeba nainstalovat Node.js a balíčkovací systém NPM. Tyto programy slouží ke spouštění JavaScriptových programů a díky nim budeme moct vytvářet moderní webové stránky. Opět následujte instrukce na [oficiální stránce](https://nodejs.org). Stáhněte a nainstalujte verzi označenou jako LTS. V současné době je to verze `18.14.1 LTS`. Číslo verze může být i vyšší, pokud tento manuál čtete někdy v budoucnosti. Pokud už máte Node.js z dřívějška, stejně si nainstalujte nejnovější verzi.
+Nyní je potřeba nainstalovat Node.js a balíčkovací systém NPM. Tyto programy slouží ke spouštění JavaScriptových programů a díky nim budeme moct vytvářet moderní webové stránky. Opět následujte instrukce na [oficiální stránce](https://nodejs.org). Stáhněte a nainstalujte verzi označenou jako LTS. V současné době je to verze `18.17.1 LTS`. Číslo verze může být i vyšší, pokud tento manuál čtete někdy v budoucnosti. Pokud už máte Node.js z dřívějška, stejně si nainstalujte nejnovější verzi.
 
-NPM je součást instalace Node.js, takže se nainstaluje společně s ním. Není ho tedy potřeba řešit zvlášť.
+NPM je součást instalace Node.js, takže se nainstaluje společně s ním. Obvykle se tedy neřeší zvlášť, my ovšem **důrazně** doporučujeme i `npm` aktualizovat na nejnovější verzi, protože se tím odstraní i jedna nepěkná chyba v aktuální instalaci Node.js. K aktualizaci budete opět potřebovat terminál nebo-li příkazovou řádku. Pokud vám nezůstal otevřený z instalace gitu, znovu ho otevřete postupem popsaným v [czechitas-podklady.cz/git-instalace/](https://czechitas-podklady.cz/git-instalace/).
+
+Do terminálu napište následující příkaz:
+
+```shell
+npm i -g npm
+```
+
+Pozor na mezery, všechna písmena jsou malá, před `g` je jedna pomlčka. Vypíše se pár řádků o tom, z jaké verze na jakou verzi se `npm` aktualizuje (skončit byste měli na verzi 9.8.1 nebo novější) a po chvilce se objeví opět řádek s výzvou (končí zobáčkem `>` ve Windows nebo dolarem `$` na MacOS a Linuxu), za kterou můžete psát další příkaz. Můžete zkusit třeba příkaz:
+
+```shell
+npm -v
```

Comment:
Na tohle je zvlášť sekce na kontrolu instalací. Má smysl to komplikovat tím, že to tu je znovu?

Reply:
Chtěl jsem tam dát něco, podle čeho poznají, že příkaz doběhl. Nenapadlo mne nic lepšího, než že můžou zadat a spustit další příkaz.

Reply:
A je ta kontrola potřeba? Mně přijde tak komplikovaná, že spíš zvyšuje riziko, že něco pokazí, ale nevím no.

Reply:
Část s `npm -v` jsem odstranil.